packages/resolve-url-loader/index.js

```diff
@@ -147,6 +147,11 @@ function resolveUrlLoader(content, sourceMap) {
 
     // prepare the adjusted sass source-map for later look-ups
     sourceMapConsumer = new SourceMapConsumer(absSourceMap);
+  } else {
+    handleAsWarning(
+      'webpack misconfiguration',
```

Comment:
Please change this to `'loader misconfiguration'` in line with other warnings.

Reply:
Sorry my mistake, you are correct that this should be `'weback misconfiguration'`. Please ignore this comment.